### PR TITLE
fix(Ziro): breaking changes for 1.1.87

### PR DESCRIPTION
**Pull request by fortoszone**
some changes i made for ziro theme with spicetify 2.10.1 and spotify 1.1.87

changelog:
- remove navbar playlist top gradient
- remove home-hover gradient
- fix tracklist card, bottom corner radius
- fix playback-bar progress-time placing

Screen Shot 2022-06-09 at 22 37 18

### CODE REVIEW

File: Ziro/user.css [hunk 1]
```diff
@@ -504,3 +508,11 @@ input:checked ~ .x-toggle-indicatorWrapper .x-toggle-indicator {
 .XmR5VFpqXOcxRvhuf6OB {
   border-radius: var(--br-1);
 }
+
+.McwcCfBLSuXa5UDU1IMw {
```
Comment (kyrie25) [inline Ziro/user.css:512]:
```suggestion
.main-rootlist-rootlistDividerContainer {
```
Class has been updated in recent CSS map change

**Review by kyrie25 — CHANGES_REQUESTED** [review]
Thanks for taking your time in making a PR!

Just a small suggestion regarding the recent CSS map updates, but are the changes still relevant/functional in the latest version of Spotify/Spicetify (1.1.88/2.10.2)?

**Comment by fortoszone** [comment]
@kyrie25 i'm already tried on spicetify 2.10.2 and spotify 1.1.88. Apparently the playlist-sidebar gonna need some adjustment. I suggest to use 1.1.87 for now. Gonna try to fix it if i have some free time. Also, thanks for the suggestion!

Screen Shot 2022-07-04 at 21 08 20

**Comment by kyrie25** [comment]
> Apparently the playlist-sidebar gonna need some adjustment. I suggest to use 1.1.87 for now. Gonna try to fix it if i have some free time.

Do you wish to merge the changes now or wait until all problems are patched? As it doesn't seem like there are any breaking changes left? Let me know by re-requesting my review whenever you think it is ready to merge :)

**Review by fortoszone — COMMENTED** [review]
> > Apparently the playlist-sidebar gonna need some adjustment. I suggest to use 1.1.87 for now. Gonna try to fix it if i have some free time.
> 
> Do you wish to merge the changes now or wait until all problems are patched? As it doesn't seem like there are any breaking changes left. Let me know by re-requesting my review whenever you think it is ready to merge :)

You can merge it for now. Thanks!

**Review by kyrie25 — APPROVED** [review]

**Pull request merged**